shared/constants/chat.js

```diff
@@ -473,6 +477,12 @@ export type SelectConversation = NoErrorTypedAction<
   'chat:selectConversation',
   {conversationIDKey: ?ConversationIDKey, fromUser: boolean}
 >
+export type SetAttachmentPlaceholderPreview = NoErrorTypedAction<
```

Comment:
This shouldn't be here any more -- added back in a merge/rebase conlict resolution?